apps/emqx_bridge_s3/src/emqx_bridge_s3_action_info.erl

```diff
@@ -0,0 +1,19 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_bridge_s3_action_info).
+
+-behaviour(emqx_action_info).
+
+-export([
+    action_type_name/0,
+    connector_type_name/0,
+    schema_module/0
+]).
+
+action_type_name() -> s3_upload.
+
+connector_type_name() -> s3.
```

Comment:
I remember there is a implicit contract with cloud UI team that action type must match connector type.

Reply:
Interesting. I'll change them to be identical, sort of curios now though: what if we want to have an application with 1 connector and, let's say, 3 different actions (e.g. simple upload, assembled upload, metadata update)?